src/pages/Leaderboard.svelte

```diff
@@ -68,20 +70,20 @@
   }
 </style>
 
-{#if leaderboard}
+{#if leaderboard_loaded}
```

Comment:
Turns out we want to track with this loading mechanism so we have data before rendering.